### PR TITLE
Edit message from oniontest1

### PR DESCRIPTION
**Pull request by oniontest1**
First pull request!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,4 +13,4 @@ This is from oniontest2. But commit shows it's from oniontest1... Be careful.
 
 Edit from oniontest1:
 ---
-I'd suggest practicing git more.
\ No newline at end of file
+I'd suggest practicing git using multiple accounts.
```
Comment (oniontest2) [inline README.md]: @oniontest1 #1 Maybe elaborate a little bit more?

File: README.md [hunk 2]
```diff
@@ -13,4 +13,4 @@ This is from oniontest2. But commit shows it's from oniontest1... Be careful.
 
 Edit from oniontest1:
```
Comment (oniontest2) [inline README.md]: This is good.

**Review by oniontest2 — COMMENTED** [review]
Please see my first review. Tnx.

File: README.md (hunk 1 above)
Comment (oniontest1) [reply README.md]: `OK`, I'll do it.

File: README.md [hunk 3]
```diff
@@ -13,4 +13,4 @@ This is from oniontest2. But commit shows it's from oniontest1... Be careful.
...
 ---
-I'd suggest practicing git using multiple accounts.
\ No newline at end of file
+I'd suggest practicing git using multiple accounts. Use different browser and use local user.name, user.email. Then add SSH Host info to `~/.ssh/config`
```
Comment (oniontest1) [inline README.md]: Detailed info for handling multiple accounts added.

Comment (oniontest2) [reply README.md]: I think it's good.

Comment (oniontest2) [reply README.md]: That looks perfect - I'm 👍 to 🚢 this!

**Pull request merged**